src/content/docs/fr/tutorial/1-setup/index.mdx

```diff
@@ -0,0 +1,71 @@
+title: 'Vérification : Unité 1 - Configuration'
+description: >-
+  Tutoriel : Construisez votre premier blog Astro —
+
+  Préparez votre environnement de développement, et créez et déployez votre premier site Astro.
+i18nReady: true
+---
+import Badge from '~/components/Badge.astro';
+import Checklist from '~/components/Checklist.astro';
+import Box from '~/components/tutorial/Box.astro';
+
+
+
+
+Maintenant que vous savez ce que vous allez créer, configurons tous les outils dont vous aurez besoin !
+
+Cette unité vous montre comment configurer votre environnement de développement et déployer sur Netlify. Passez directement à l'[Unité 2](/fr/tutorial/2-pages/) si vous êtes déjà à l'aise avec votre environnement et votre flux de travail.
+
+:::tip[utilisation de StackBlitz]
+
+Vous voulez compléter ce tutoriel dans un éditeur de code en ligne ?
+<details>
+<summary>Suivez ces instructions, puis passez directement à l'Unité 2 !</summary>
+
+**Configurer StackBlitz**
+1. Rendez-vous sur [astro.new](https://astro.new) et cliquez sur le bouton pour ouvrir le modèle "Empty Project" dans StackBlitz.
+
+2. Cliquez sur "Sign in" en haut à droite pour vous connecter avec vos informations GitHub.
+
+3. En haut à gauche de la fenêtre d'édition StackBlitz, cliquez pour "fork" le modèle (l'enregistrer sur le tableau de bord de votre compte).
+
+4. Attendez que le projet se charge, et vous verrez un aperçu en direct du modèle "Empty Project".
+
+**Effectuer un changement**
+
+Dans le volet des fichiers, vous devriez voir `src/pages/index.astro`. Cliquez pour l'ouvrir, et suivez [Écrivez votre première ligne Astro](/fr/tutorial/1-setup/3/) pour effectuer un changement sur ce fichier.
+
+**Créer un dépôt GitHub**
+
+1. Appuyez sur le bouton <kbd>Connect Repository</kbd> en haut de votre liste de fichiers, saisissez un nouveau nom pour votre dépôt, et cliquez sur <kbd>Create repo & push</kbd>.
+
+2. Lorsque vous avez des modifications à valider dans GitHub, un bouton "Commit" apparaîtra en haut à gauche de votre espace de travail. En cliquant dessus, vous pourrez saisir un message de validation et mettre à jour votre dépôt.
+
+**Déployer votre site**
+
+Si vous souhaitez déployer sur Netlify, passez à [Déployer votre site sur le web](/fr/tutorial/1-setup/5/).
+Sinon, passez à l'[Unité 2](/fr/tutorial/2-pages/) pour commencer à créer avec Astro !
+
+</details>
+:::
+
+## Où allons-nous ?
+
+Dans cette unité, vous allez **créer un nouveau projet** qui est **stocké en ligne sur GitHub** et **connecté à Netlify**.
+
+En écrivant du code, vous allez périodiquement valider vos modifications dans GitHub. Netlify utilisera les fichiers de votre dépôt GitHub pour construire votre site web, puis le publier sur Internet à une adresse unique où n'importe qui peut le consulter.
+
+Chaque fois que vous validez une modification dans GitHub, une notification sera envoyée à Netlify. Ensuite, Netlify reconstruira automatiquement votre site pour refléter ces modifications.
```

Comment:
```suggestion
Chaque fois que vous validez une modification dans GitHub, une notification sera envoyée à Netlify. Ensuite, Netlify reconstruira et republiera automatiquement votre site pour refléter ces modifications.
```

Reply:
Les deux propositions sont correct, ma préférence irait pour la proposition d'origine

Both proposals are correct, but my preference would be for the original one.

LGTM